enterprise/app/models/enterprise/audit/user.rb

```diff
@@ -0,0 +1,30 @@
+module Enterprise::Audit::User
+  extend ActiveSupport::Concern
+
+  included do
+    audited except: [
```

Comment:
may be use `only` here so that we selectively add new fields ensuring nothing sensitive will be added accidently ?

Reply:
done